File: Assignment_6/program5.c
#include <stdio.h>

int Table(int iNo)
{
   int  iCnt = 0 ;
   int  iMulti = 1;

   if (iNo < 0)
   {
        iNo = -iNo ;
   }
   
   for(iCnt =10 ; iCnt >=1 ; iCnt --)
   {         
        iMulti = iNo*iCnt ;
        printf("%d\t",iMulti);
   }
     return iNo ;
}

int main()
{
    int iValue = 0;

    printf("Enter The Number:");
    scanf("%d",&iValue);

    Table(iValue);

    return 0;
}

File: Assignment_17/program5.java
import java.util.Scanner ; 

class Logic
{
    void  Table(int iNo)
    {
        int i = 0 ;
        int iMult = 0 ; 
        
        for(i = 1 ; i < 11 ; i++)
        {
            iMult = iNo * i ;
            System.out.println(+iMult);
        }
        
    }
}

class program5
{
    public static void main(String A[]) 
    {
        int iValue = 0  ; 
        
        Scanner sobj = new Scanner(System.in); 
        
        System.out.println("Enter the First Number :");
        iValue= sobj.nextInt() ; 

        System.out.println("Table of "+iValue);
        Logic lobj = new Logic();
        lobj.Table(iValue);
       
        sobj.close();
    }
}